package.json

```diff
@@ -38,6 +38,7 @@
     "abort-controller": "^2.0.3",
     "ascii-table": "0.0.9",
     "axios": "^0.20.0",
+    "brainfuck-node": "^1.0.2",
```

Comment:
🤔

Reply:
![](https://i.imgur.com/Mi7GN0t.png)

https://github.com/SwitchbladeBot/switchblade/pull/1226/files

they not added the module itself, that's why i added it